fml/message_loop.h

```diff
@@ -15,7 +15,6 @@ class MessageLoopImpl;
 
 class MessageLoop {
  public:
-  FML_EMBEDDER_ONLY
```

Comment:
can this cause issues @chinmaygarde ?

Reply:
This is to prevent the accidental use of fml::MessageLoop::GetCurrent() in a unit-test. The current message loop cannot be accessed from the cross platform engine components because message loop management is an embedder concern. But, if you want to use in in a unit-test, you can `#define FML_USED_ON_EMBEDDER` in your unit-test before including any other files.

But, you shouldn't have to do this in the first place. Instead of using raw threads, use fml::Thread which have their own task runners that you can access without resorting to removing this sanity check.

Reply:
TL;DR: Please don't remove this and use fml::Thread with the GetTaskRunner call on that thread to access task runners.